### ISSUE TITLE
Dropdown - _this._dropdown.blur is not a function

### ISSUE THREAD

**Issue by giacomorebonato** [Semantic-Org/Semantic-UI-React#695]
I am using a dropdown inside a Menu as a Menu.Item  

``` js
<Menu.Menu position='right'>
    <Dropdown as={Menu.Item} text='Profile'>
        <Dropdown.Menu>
            <Dropdown.Item>Electronics</Dropdown.Item>
            <Dropdown.Item>Automotive</Dropdown.Item>
            <Dropdown.Item>Home</Dropdown.Item>
        </Dropdown.Menu>
    </Dropdown>
</Menu.Menu>
```
### Steps
1. I click to open the dropdown
2. I click outside the dropdown
### Actual Result

I get the following error in the console when the dropdown closes and when I unsuccessfully try to re-open it  

```
Dropdown.js?ea24:517 Uncaught TypeError: _this._dropdown.blur is not a function
```
### Version

0.56.4
### Testcase

http://codepen.io/giacomorebonato/pen/bwmNbm

**Comment by levithomason** [comment]
Duplicate https://github.com/Semantic-Org/Semantic-UI-React/pull/659

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.64.4`.
